lib/engines/content_block_manager/app/views/content_block_manager/content_block/documents/show.html.erb

```diff
@@ -45,7 +45,7 @@
 <div class="govuk-grid-row">
   <div class="govuk-grid-column-full">
     <%= render(
-          ContentBlockManager::ContentBlock::Document::Show::SummaryListComponent.new(
+          ContentBlockManager::ContentBlock::Document::Show::SummaryCardComponent.new(
```

Comment:
I'm not yet deleting the SummaryListComponent in `Show` because the SummaryCardComponent in `Index` inherits from it and there's a fair bit to unpick - think that makes sense to do when we go to re-design the `Index` view, but could pick it up here if we think we need to.